src/main/java/com/scouts/kitchenplanerbackend/repositories/projects/ProjectRepository.java

```diff
@@ -104,4 +110,23 @@ void updateMetaData(@Param("name") String name, @Param("startDate") Date startDa
     @Modifying(clearAutomatically = true)
     @Query("update ProjectEntity p set p.imageVersion = (p.imageVersion +1) where p.id = :id")
     void increaseImageVersionById(@Param("id") Long id);
+
+    /**
+     * Updates the path to a project's image
+     * @param imageUri The new path
```

Comment:
Make them privat/protected or something like that, because the uri only should be updated through "updateImagePath"